src/libs/PolicyUtils.ts

```diff
@@ -117,6 +117,16 @@ const isFreeGroupPolicy = (policy: OnyxEntry<Policy> | EmptyObject): boolean =>
 
 const isPolicyMember = (policyID: string, policies: OnyxCollection<Policy>): boolean => Object.values(policies ?? {}).some((policy) => policy?.id === policyID);
 
+const isPolicyPayer = (policy: OnyxEntry<Policy> | EmptyObject, session: OnyxEntry<Session>, isApproved: boolean, isManager: boolean, isAdmin: boolean): boolean => {
+    if (policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES) {
+        const isReimburser = session?.email === policy?.reimburserEmail;
+        return isReimburser && (isApproved || isManager);
+    }
+    if (policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL) {
+        return isAdmin && (isApproved || isManager);
+    }
+    return false;
```

Comment:
In order to show the Pay button for IOUs (and free workspaces for now), I think we need to check if the user is manager and if the policy is not a group policy.

Reply:
@marcochavezf 

Thank you! Focused too much only on policies and let this blunder slide. Fixed now.

One last question, we show only `Pay elsewhere` for free workspaces, right? Or do we show `Pay with Expensify` as well?

Reply:
Oh good question, we should show both `Pay elsewhere` and `Pay with Expensify` for non-group policies (free workspaces and IOUs). In theory for free workspaces the admin and the manager should be the same, so we should probably show both buttons to the manager if policy is a non-group policy/workspace.

Reply:
Thanks. Done. 👍